projects/Morse Code Encoder & Decoder/morse_code.py

```diff
@@ -0,0 +1,69 @@
+codes = {
```

Comment:
Morse code alphabet Dictionary

Reply:
Should I add a comment stating that the dictionary represents plain text and their morse code values? Plus I am new to contributing so it would be nice to give me step-by-step instructions.
Thank you

Reply:
It's not necessary to add comments for little projects.